js/offers-markup.js

```diff
@@ -0,0 +1,67 @@
+const typesView = {
+  'flat': 'Квартира',
+  'bungalow': 'Бунгало',
+  'house': 'Дом',
+  'palace': 'Дворец',
+  'hotel': 'Отель'
+};
+
+const roomsLabel = (roomsCount) => {
+  if (roomsCount === 1) return 'комната';
+  return roomsCount <= 4 ? 'комнаты' : 'комнат';
+}
+
+const mapCanvas = document.querySelector('#map-canvas');
+
+const similarOfferTemplate = document.querySelector('#card')
+  .content
+  .querySelector('.popup');
+
+const renderOffers = (offers) => {
+
+  const similarListFragment = document.createDocumentFragment();
+
+  offers.forEach(({offer}) => {
+
+    const offerElement = similarOfferTemplate.cloneNode(true);
+
+    offerElement.querySelector('.popup__title').textContent = offer.title;
+    offerElement.querySelector('.popup__text--address').textContent = offer.address;
+    offerElement.querySelector('.popup__text--price').textContent = `${offer.price} ₽/ночь`;
+    offerElement.querySelector('.popup__type').textContent = typesView[offer.type];
+    offerElement.querySelector('.popup__text--capacity').textContent = `${offer.rooms} ${roomsLabel(offer.rooms)} для ${offer.guests} гостей`;
+
+    const elementDescription = offerElement.querySelector('.popup__description');
+    if (offer.description === '') {
+      elementDescription.textContent = '';
+      elementDescription.classList.add('hidden');
```

Comment:
```suggestion
      elementDescription.hidden = true;
```
можешь его сразу скрывать, тк он больше не нужен)